src/V3AstNodeOther.h

```diff
@@ -2851,6 +2851,16 @@ class AstCMethodHard final : public AstNodeStmt {
         statement(true);
         dtypeSetVoid();
     }
+    int instrCount() const override {
+        if (AstBasicDType* const basicp = fromp()->dtypep()->basicp()) {
+            if (basicp->isTriggerVec() && m_name == "at") {
```

Comment:
Rather than hardcoding string matches, I'd prefer we have an accessor that stores the cost, and set it when we use "at", also then setting this properly considered when we construct the nodes. 

This can wait and add a TODO if you prefer.

Reply:
I agree with you the string matching is not ideal, but I think setting the cost explicitly is not particularly helpful at the moment. All uses of "at" would have the same cost so it's just boilerplate at every site that might or might not be forgotten. Long term it would be nice to have a more structured description of the hardcoded library methods, including signature, which could also incorporate a default cost, so this issue does not occur on a case by case basis but is handled uniformly. I suggest I will file an issue to do that and add the TODO for now as you recommend.